File: Easy/Java/Merge_Sorted_Array(88).java
class Solution {
public void merge(int[] nums1, int m, int[] nums2, int n) {
    if(n==0) return;
     m=nums1.length;
    n=n-1;
    for(int i=m-1;i>=0;i--){
        if(nums1[i]==0){
            nums1[i]=nums2[n];
            
        }
        if(n==0){
            break;
        }
        n--;
    }
    Arrays.sort(nums1);
}
}

File: Easy/C++/Merge_Sorted_Array(88).cpp
class Solution {
public:
    void merge(vector<int>& nums1, int m, vector<int>& nums2, int n) {
        nums1.insert(nums1.begin()+m,nums2.begin(),nums2.end());
        for(int i=0;i<n;i++)
            nums1.pop_back();
        sort(nums1.begin(),nums1.end());
    }
};